### PR TITLE
Update ar.json

### PR DESCRIPTION
**Pull request by ghost**
Adding more translations (updating the missing non translated strings) and some little minimal corrections.

### CODE REVIEW

File: app/javascript/mastodon/locales/ar.json [hunk 1]
```diff
@@ -164,4 +164,4 @@
   "video_player.toggle_sound": "تبديل الصوت",
   "video_player.toggle_visible": "إظهار / إخفاء الفيديو",
   "video_player.video_error": "تعذر تشغيل الفيديو"
-}
\ No newline at end of file
+}
```
Comment (ykzts) [inline app/javascript/mastodon/locales/ar.json]: Delete final newline.

**Review by ykzts — CHANGES_REQUESTED** [review]

File: app/javascript/mastodon/locales/ar.json (hunk 1 above)
Comment (BoFFire) [inline app/javascript/mastodon/locales/ar.json]: Okay.

**Review by ykzts — APPROVED** [review]

File: app/javascript/mastodon/locales/ar.json (hunk 1 above)
Comment (BoFFire) [inline app/javascript/mastodon/locales/ar.json]: Are you sure ? @ykzts ?

**Comment by BoFFire** [comment]
Closing this pull request because of a } last line. waiting for confirmation.

**Comment by ykzts** [comment]
oops. sorry

File: app/javascript/mastodon/locales/ar.json (hunk 1 above)
Comment (ykzts) [inline app/javascript/mastodon/locales/ar.json]: Please only delete the last newline. } Does not have to be deleted. Please make sure that there is no difference in the last line.

**Review by ykzts — CHANGES_REQUESTED** [review]
please retry.

File: app/javascript/mastodon/locales/ar.json (hunk 1 above)
Comment (BoFFire) [inline app/javascript/mastodon/locales/ar.json]: :)Thank you.

Comment (ykzts) [inline app/javascript/mastodon/locales/ar.json]: Can you run 'yarn run manage: translations'?

Comment (BoFFire) [inline app/javascript/mastodon/locales/ar.json]: I don't know what yarn is ;)

Comment (ykzts) [inline app/javascript/mastodon/locales/ar.json]: realy!? So how do you check the translation?

**Review by ykzts — APPROVED** [review]
Line breaks on the last line are only non-essential differences. So there is no problem if you approve it.

File: app/javascript/mastodon/locales/ar.json (hunk 1 above)
Comment (BoFFire) [inline app/javascript/mastodon/locales/ar.json]: Manually ;-)

**Pull request merged**